src/css/Footer.scss

```diff
@@ -1,10 +1,38 @@
+@import '~bootstrap/scss/functions';
+@import '~bootstrap/scss/variables';
+@import '~bootstrap/scss/mixins';
+
+@include media-breakpoint-down(md) {
+  footer {
+    display: none;
+  }
+}
+
+
 footer {
-  margin-top: 30px;
-  padding-top: 30px;
   width: 100%;
   background: linear-gradient(#1A1A1A, #121212);
 
   p {
     text-align: right;
   }
 }
+
+.footer-list{
+  display: flex;
+  flex-direction: row;
+  left: -50px;
+  position: relative;
+}
+
+.footer-list > li{
+  list-style: none;
+  margin: 0px 50px 0px 50px;
+}
+
+@media screen and (max-width: 1350px) {
+  .footer-list{
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr;
```

Comment:
`grid-template-columns` is not supported in IE. does autoprefixer cover us here? I can test this out in IE if need be.

Reply:
I'll try to format it differently then without using grid.

Reply:
Unless there is a different way to make it work, I'm not familiar with autoprefixer

Reply:
The fallback property is -ms-grid-columns: 1fr 1fr 1fr; if you could test it it would be great.

Reply:
Footer looks fine in IE.

Today I learned that Stream page is actually just broken on IE today anyways (without this change), so maybe I should just remove that browser from [Supported Browsers](https://github.com/MemeLabs/Rustla2/wiki/Browser-Support) 😏